src/components/fields/date-input/date-input.tsx

```diff
@@ -121,16 +112,27 @@ export const DateInput = (props: IGenericFieldProps<IDateInputSchema>) => {
 		} else {
```

Comment:
In line 108 
```
} else if (!day.length || !month.length || year.length !== 4) {
```
I am wondering if you should check if the day and month length <= 2 instead for more accuracy?